Mage/src/main/java/mage/abilities/effects/common/CreateTokenCopyTargetEffect.java

```diff
@@ -154,6 +154,7 @@ protected CreateTokenCopyTargetEffect(final CreateTokenCopyTargetEffect effect)
         this.tokenPower = effect.tokenPower;
         this.tokenToughness = effect.tokenToughness;
         this.useLKI = effect.useLKI;
+        this.permanentModifier = effect.permanentModifier;
```

Comment:
please update TODO comment for the field, to remove the part that is no longer relevant after this fix